model/context.go

```diff
@@ -0,0 +1,79 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package model
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/elastic/apm-server/utility"
+	"github.com/elastic/beats/libbeat/common"
+)
+
+func HttpFields(http common.MapStr) common.MapStr {
+	var source = func(dottedKeys string) interface{} {
+		return utility.Get(http, dottedKeys)
+	}
+
+	destination := common.MapStr{}
+	utility.Add(destination, "version", source("request.http_version"))
+	utility.DeepAdd(destination, "request.method", source("request.method"))
+
+	utility.DeepAdd(destination, "request.method", source("request.method"))
```

Comment:
This is duplicated. 

> Http request method.
The field value must be normalized to lowercase for querying. See "Lowercase Capitalization" in the "Implementing ECS" section.

Referring to [ECS](https://github.com/elastic/ecs#-http-fields) the `http.request.method` needs to be lowercase.

Reply:
Another good catch, implemented in a56d3f5ba827a8625283b1b56916ee298c37d8e0